### DISCUSSION TITLE
Polar Stereographic projection

### DISCUSSION THREAD

**Discussion by Tom-Tyburczy** [DOI-USGS/usgscsm#356] [Q&A]
Hi all! 

We at LROC are testing some images we have of Shackleton Ridge at the Lunar south pole. 

We've been testing this using the NAC pair:  M139797542/M139811097 
Image .cubs and their .json files can be downloaded [here](https://drops.ser.asu.edu/download?cookie=3bd7b929aa&filename=ShackRidge.zip&uploaded_at=1643072274&user_id=ttyburczy) (Link expires Feb 7, 2022)

We are having some issues with triangulation; manual tie points somehow end up with null coordinates, and trying to run APM crashes GXP. Can the usgscsm be used for polar images?

Additonally, we've been able to generate terrain of Shackleton Ridge using SOCET SET 5.6 which has a 'Polar Stereographic' projection, but we're having trouble finding such a projection in GXP. Does anyone know if there is a Polar Stereographic projection option in GXP?

We are using SOCET GXP version  4.5.0.1.232, and usgscsm 0.1.436

Thank you!

**Comment by jessemapel** [comment]
USGSCSM doesn't use projections at all, that's entirely within GXP. There could be an issue with the camera model and those images though. Particularly the part about points getting null coordinates concerns me.

I'll take a look at the images your provided.

**Reply by meganhenriksen** [reply]
We found the config files in the GXP setup and managed to implement this.
